extensions/schemas/standard/clause_0_front_material.adoc

```diff
@@ -1,37 +1,37 @@
 [big]*i.     Abstract*
 
-OGC API standards define modular API building blocks to spatially enable Web APIs in a consistent way. The <<OpenAPI,OpenAPI specification>> is used to define the API building blocks.
+OGC API Standards define modular API building blocks to spatially enable Web API implementations in a consistent way. The <<OpenAPI,OpenAPI specification>> is used to define the API building blocks.
 
-OGC API Features provides API building blocks to create, modify and query features on the Web. OGC API Features is comprised of multiple parts. Each part is a separate standard.
+OGC API - Features Standard defines building blocks to create, modify and query features on the Web. The OGC API - Features Standard is comprised of multiple parts. Each part is a separate standard.
 
-This Standard specifies how geospatial data such as features can be described by a logical schema and how such schemas are published in an OGC Web API.
+The OGC API - Features - Part 5: Schemas Standard (hereafter also referred to as "this Standard" or "this document") specifies how geospatial data such as features can be described by a logical schema and how such schemas are published in an OGC Web API implementation.
 
-As logical schemas, the schemas describe the data independent of a particular format to represent the data. 
+As logical schemas, the schemas describe the data independent of a particular format and encoding to represent the data. 
```

Comment:
"format and encoding" is redundant. These are basically used as synonyms. ISO 19118 has introduced "encoding" while "format" is the more commonly used term. I think what we should do is to use only a single term in the document and to use it consistently.